tests/sync/app.cpp

```diff
@@ -1544,7 +1646,10 @@ static const std::string good_access_token2 =  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpX
 std::string UnitTestTransport::access_token = good_access_token;
 
 static const std::string bad_access_token = "lolwut";
+static const std::string dummy_device_id = "123400000000000000000000";
+static const std::string app_name = "django";
 
+const std::string UnitTestTransport::app_name = "django";
```

Comment:
why do we need two app_name variables with the same values?

Reply:
Removed the secondary app_name